.github/workflows/flutter_package.yml

```diff
@@ -43,6 +43,10 @@ on:
         required: false
         type: boolean
         default: false
+      very_good_cli_version:
+        required: false
+        type: string
+        default: "0.11.0+1"
```

Comment:
Shouldn't this default to empty? Then the behavior would be analogous to the "fluttter_version" parameter.